### PR TITLE
Added crop_tools.json and updated mathematical hoes with missing hoes

### PR DESCRIPTION
**Pull request by Su386yt**

### CODE REVIEW

File: data/constants/crop_tools.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+    "tile.crops" : {
+        "math" : [
+            "THEORETICAL_HOE_WHEAT_1",
+            "THEORETICAL_HOE_WHEAT_2",
+            "THEORETICAL_HOE_WHEAT_3"
+        ],
+        "other_skyblock" : [
+            "ROOKIE_HOE"
```
Comment (j10a1n15) [inline data/constants/crop_tools.json]: Add (advanced) gardening hoe

File: data/constants/crop_tools.json [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    },
+    "tile.nether_stalk" : {
+        "math" : [
+            "THEORETICAL_HOE_WARTS_1",
+            "THEORETICAL_HOE_WARTS_2",
+            "THEORETICAL_HOE_WARTS_3"
+        ],
+        "other_skyblock" : [
+            "ROOKIE_HOE"
+        ],
```
Comment (j10a1n15) [inline data/constants/crop_tools.json:92]: Add (Advanced) gardening axe

File: data/constants/crop_tools.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        "other_skyblock" : [
+            "TREECAPITATOR_AXE"
+        ],
+        "other_minecraft" : [
+            "minecraft:golden_axe"
+        ]
+    },
+    "tile.melon" : {
+        "math" : [
+            "MELON_DICER",
```
Comment (j10a1n15) [inline data/constants/crop_tools.json:120]: Add deadelous axe

File: data/constants/mathematical_hoes.json [hunk 4]
```diff
@@ -15,11 +15,15 @@
...
+        "THEORETICAL_HOE_WARTS_2",
+        "THEORETICAL_HOE_WARTS_3",
         "PUMPKIN_DICER",
         "PUMPKIN_DICER_2",
         "PUMPKIN_DICER_3",
         "MELON_DICER",
         "MELON_DICER_2",
-        "MELON_DICER_3"
+        "MELON_DICER_3",
+        "FUNGI_CUTTER"
```
Comment (j10a1n15) [inline data/constants/mathematical_hoes.json]: Please remove the cutter, since its rc is an ability

**Review by j10a1n15 — CHANGES_REQUESTED** [review]

File: data/constants/crop_tools.json (hunk 3 above)
Comment (Su386yt) [reply data/constants/crop_tools.json:120]: Why?

Comment (j10a1n15) [reply data/constants/crop_tools.json:120]: Gives lot of strength, which makes it good for mush cow

Comment (Su386yt) [reply data/constants/crop_tools.json:120]: Done

File: data/constants/mathematical_hoes.json [hunk 5]
```diff
@@ -15,6 +15,9 @@
         "THEORETICAL_HOE_CARROT_1",
         "THEORETICAL_HOE_CARROT_2",
         "THEORETICAL_HOE_CARROT_3",
+        "THEORETICAL_HOE_WARTS_1",
+        "THEORETICAL_HOE_WARTS_2",
+        "THEORETICAL_HOE_WARTS_3",
```
Comment (j10a1n15) [inline data/constants/mathematical_hoes.json:20]: This list is for the block clicks right?
Why do we block clicks on t3 hoes

**Review by j10a1n15 — APPROVED** [review]

File: data/constants/mathematical_hoes.json (hunk 5 above)
Comment (Su386yt) [reply data/constants/mathematical_hoes.json:20]: No idea tbh

Comment (j10a1n15) [reply data/constants/mathematical_hoes.json:20]: Also file name is a bit misleading honestly

Comment (Su386yt) [reply data/constants/mathematical_hoes.json:20]: which one

Comment (j10a1n15) [reply data/constants/mathematical_hoes.json:20]: Should rather be smth like "block_hoe_clicks.json" or smth

Comment (Su386yt) [reply data/constants/mathematical_hoes.json:20]: I mean tbh it's literally a file with mathematical hoes

**Pull request merged**